happybase/batch.py

```diff
@@ -45,7 +45,7 @@ def _reset_mutations(self):
 
     def send(self):
         """Send the batch to the server."""
-        bms = [BatchMutation(row, m) for row, m in self._mutations.iteritems()]
+        bms = [BatchMutation(row, m) for row, m in self._mutations.items()]
```

Comment:
this makes copies into a new list which iterated and then thrown away in python 2 :(